docs/book/src/migration/v1vsv2.md

```diff
@@ -0,0 +1,59 @@
+# Kubebuilder v1 vs v2
+
+This document should cover all breaking changes when migrating from v1 to v2.
+
+The details of all changes can be found in
+[controller-runtime](https://github.com/kubernetes-sigs/controller-runtime/releases),
+[controller-tools](https://github.com/kubernetes-sigs/controller-tools/releases)
+and [kubebuilder](https://github.com/kubernetes-sigs/kubebuilder/releases)
+release notes.
+
+## Common changes
+
+- All three projects now use `go modules` instead of `dep`.
+
+## controller-runtime
+
+- `client.List` now uses functional options `List(ctx, list, ...option)` instead
+of `List(ctx, ListOptions, list)`.
+- `client.DeleteAllOf` was added.
+
+- Metrics are on by default now.
+
+- A number of packages under `pkg/runtime` have been moved, with their old
+locations deprecated. See the [godocs][pkg-runtime-godoc] for more information.
+
+#### Webhook-related
+
+- Automatic certificate generation for webhooks has been removed, and webhooks
+will no longer self-register. Use controller-tools to generate a webhook
+configuration. If you need certificate generation, we recommend using
+[cert-manager](https://github.com/jetstack/cert-manager). Kubebuilder v2 will
+scaffold out cert manager configs for you to use -- see the
+[Webhook Tutorial](/cronjob-tutorial/webhook-implementation.md) for more details.
+
+- The `builder` package now has separate builders for controllers and webhooks.
+
+## controller-tools
+
+The generator framework has been rewritten in v2. It still works the same as
+before in many cases. But be aware that there are some breaking changes.
+Please check [marker document](/reference/markers.md) for more details.
+
+## kubebuilder
+
+- Kubebuilder v2 introduces a simplified project layout. You can find the design
+doc [here](https://github.com/kubernetes-sigs/kubebuilder/blob/master/designs/simplified-scaffolding.md).
+
+- In v1, the manager is deployed as `StatefulSet`, while as `Deployment` in v2.
+
+- Add the `kubebuilder create webhook` command to scaffold
```

Comment:
this construction makes it seem like you're telling the user to do something,  Use something like this instead:

```suggestion
- The `kubebuilder create webhook` command was added to scaffold
```